### PR TITLE
Fix hass setting on stack

### PR DESCRIPTION
**Pull request by balloob**
No longer set `hass` on a card in a stack if it's undefined.

### CODE REVIEW

File: src/panels/lovelace/cards/hui-vertical-stack-card.ts [hunk 1]
```diff
@@ -1,38 +1,11 @@
...
-
-    if (!this._cards) {
-      return;
-    }
-
-    for (const element of this._cards) {
-      element.hass = this._hass;
-    }
-  }
+import { HuiStackCard } from "./hui-stack-card.js";
```
Comment (thomasloven) [inline src/panels/lovelace/cards/hui-vertical-stack-card.ts]: Remove the .js

**Review by thomasloven — DISMISSED** [review]

File: src/panels/lovelace/cards/hui-vertical-stack-card.ts (hunk 1 above)
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-vertical-stack-card.ts]: This needs to be `import { HuiStackCard } from "./hui-stack-card";`

Comment (zsarnett) [reply src/panels/lovelace/cards/hui-vertical-stack-card.ts]: Thomas is faster than I

**Comment by exetico** [comment]
Could this be a problem with `vertical-stack` too?
https://community.home-assistant.io/t/0-81-new-lovelace-cards-lg-soundbars-dyson-fans-elk-m1/75277/92?u=exetico

```
hui-conditional-card TypeError: "hass is undefined"
	hass https://mydomain.com/local/mini-media-player/mini-media-player.js:20:11
	setConfig https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:418:1949
	create_card_element_createElement https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1933:2748
	createCardElement https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1933:3897
	_cards https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1131:1071
	setConfig https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1131:1033
	create_card_element_createElement https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1933:2748
	createCardElement https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1933:3897
	elements https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1995:1068
	_createCards https://mydomain.com/frontend_latest/8avalueremovedd749b3.chunk.js:1995:1024
8avalueremovedd749b3.chunk.js:1933
```

**Comment by iantrich** [comment]
@exetico fixed in https://github.com/home-assistant/home-assistant-polymer/pull/1868 for stacks

**Pull request merged**